src/components/Datasets.jsx

```diff
@@ -63,6 +72,19 @@ class Datasets extends React.Component {
       });
   }
 
+  handleSortBy(val) {
+    let sortby = val.value;
+    let items = this.state.items;
```

Comment:
Can you do this, so we don't mutate the array (calling `items.sort` mutates the array in-place, and you don't want to directly mutate state -- see https://stackoverflow.com/questions/9592740/how-can-you-sort-an-array-without-mutating-the-original-array):

```suggestion
    let items = [...this.state.items];
```

Reply:
Fixed! Needed to alter some variable names since the page displays `this.state.filteredItems`.